app/assets/stylesheets/frontend/base.scss

```diff
@@ -9,18 +9,11 @@ $govuk-new-link-styles: true;
 
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/big-number";
-@import "govuk_publishing_components/components/button";
-@import "govuk_publishing_components/components/breadcrumbs";
 @import "govuk_publishing_components/components/contents-list";
 @import "govuk_publishing_components/components/document-list";
-@import "govuk_publishing_components/components/error-message";
-@import "govuk_publishing_components/components/feedback";
+@import "govuk_publishing_components/components/details";
 @import "govuk_publishing_components/components/govspeak";
-@import "govuk_publishing_components/components/heading";
```

Comment:
Heading is also referenced in `html-publication.scss`. Actually a whole load of things are still referenced there so I'll stop pointing them out individually and let you look into it.